crypto/cipher/aes_icm.c

```diff
@@ -198,21 +200,22 @@ static srtp_err_status_t srtp_aes_icm_context_init (void *cv, const uint8_t *key
     v128_set_to_zero(&c->offset);
 
     copy_len = c->key_size - base_key_len;
-    /* force last two octets of the offset to be left zero (for srtp compatibility) */
+    /* force last two octets of the offset to be left zero (for srtp
+     * compatibility) */
     if (copy_len > SRTP_SALT_LEN) {
         copy_len = SRTP_SALT_LEN;
     }
 
     memcpy(&c->counter, key + base_key_len, copy_len);
     memcpy(&c->offset, key + base_key_len, copy_len);
 
-    debug_print(srtp_mod_aes_icm,
-                "key:  %s", srtp_octet_string_hex_string(key, base_key_len));
-    debug_print(srtp_mod_aes_icm,
-                "offset: %s", v128_hex_string(&c->offset));
+    debug_print(srtp_mod_aes_icm, "key:  %s",
+                srtp_octet_string_hex_string(key, base_key_len));
+    debug_print(srtp_mod_aes_icm, "offset: %s", v128_hex_string(&c->offset));
 
     /* expand key */
-    status = srtp_aes_expand_encryption_key(key, base_key_len, &c->expanded_key);
+    status =
+        srtp_aes_expand_encryption_key(key, base_key_len, &c->expanded_key);
```

Comment:
Not sure about this one, would have thought the assignment should stay on the same line but the function arguments break. @thisisG  what do you think?

Reply:
We can either play around with

PenaltyBreakAssignment (unsigned)
and
PenaltyBreakBeforeFirstCallParameter (unsigned)

or we can accept this as a quirk.

I am fine with either, let me know if we should have a stab at fixing it!

Reply:
Personally, I like the way it turned out.

Reply:
It is not my personal preference but if it is not to common then it is ok. Will keep it like this for now and we can see what happens with the rest.